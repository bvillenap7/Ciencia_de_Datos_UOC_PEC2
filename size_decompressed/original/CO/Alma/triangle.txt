Massive triangle over NORAD for 4 hrs today!;2014-10-03 07:00:00;4 hours;Massive triangle over NORAD for 4 hrs today! Walking to the local store at 7am, I noticed 3 bright lights hanging in the sky.  They looked like stars in a perfect triangle and didn't move for four hours.   This was nothing man made, it was bigger than a football field and after the first hour military aircraft started flying all under and around it.   It looked to be over NORAD then it dissapeared one light at a time. The military jets are still searching the sky at 6pm.;39.2871;-106.0632
