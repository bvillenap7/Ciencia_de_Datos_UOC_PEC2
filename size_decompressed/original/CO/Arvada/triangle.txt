HUGE Triangle UFO spotted over Denver;2001-08-12 09:20:00;2 Minutes;HUGE Triangle UFO spotted over Denver In this event, I was shocked to see this object.  At least a couple of football fields long, made no sound whatsoever.  It was around 9:20 Mountain Daylight Time.  August 12 of 2001.  I was so extatic.  I rushed into my house to grab the digital camara.  I also was able to take several pictures of this craft. At first, I thought it was two seperate craft with bright red pulsating lights.  However, they were in syncronization with each other.  So therefore it had to be one craft.  They were coming from the direction of Rocky Flatts (nuclear facility) and also our local small airport here.  It was moving from the NW to the SE in the direction of Downtown Denver so I'm hoping a lot more people would have seen this object.  If you have any questions, my email is ((e-address deleted))  you may request photos of this sighting. ((NUFORC Note:  We will request a copy of the photograph.  It is not clear whether the sighting occurred during daytime, or at night.  We will attempt to clarify this point with the witness.  PD));39.8289982352941;-105.110888235294
silent v shape  craft , no light's, moving west to east, circles on bottom from center and two down the wings;2008-12-23 12:00:00;9 seconds;silent v shape  craft , no light's, moving west to east, circles on bottom from center and two down the wings 12/23/08- snow was on the ground and a clear full moon night.  Looked up and saw a boomerrang like craft flying from west to east.   Waited for a sound but only silent night surrounded, no lights whatsoever.  V shape and with dark circles on bottom, from center and two on wing sides.  Moving rather fast and only got a glimpse of it.  High enough in sky but low enough to catch reflection of moon light off the snow.   Again, on 08/15/09 saw the object flying westerly at same speed at 3:43 a.m. both nights a vivid array of shooting star activity..  Too dark to capture a picture, too fast to run and get my cheap camera phone to try.  I often wonder of whether this craft is of this region in the mountains, my instinct tells me that it is .perhaps.  Will keep the eye’s open.  You too.;39.8289982352941;-105.110888235294
A triangular shaped object this a white light at each vertex and a red light in the center.;2011-11-09 18:40:00;about 5 minutes;A triangular shaped object this a white light at each vertex and a red light in the center. On November 9th 2011, my brother and I ware driving down 80th Avenue going to our local church for a confirmation meeting at about 6:40pm.  My brother was driving. He looked to his left and said, What’s with the lights? I didn’t see them at the time because of the roof of the car.  I thought it was a plane or a street light or something. Then we pull up to the next intersection and he points it out again.  This time I see a vertical line of four lights.  One was red.  We turn onto the street where the church is off of and we get a better look at it and notice that it’s triangular shaped.  And the red light was in the middle.  And we also notice that it’s spinning slowly and moving.  We were basically following it though the parking lot of the church until we parked at one end of the parking lot.  The object was spinning slowly as it moved across the sky at about 30 or 40 mph. We get out of the car and watch it continue moving in the same direction (East) until it was too far away to see.  We also notice, after getting out of the car, that it was completely silent.  It was probably the most extraordinary things I have ever seen.;39.8289982352941;-105.110888235294
Red Orbs.;2014-02-15 18:14:00;2 minutes;Red Orbs Witnessed three small Red Copper Orbs in formation.  Appeared and lasted about two minutes.   As we were coming in to the house, we saw lights pop on and formed a triangle pattern.  As we watched the objects pulsate, one began to slowly fade out in to a small copper dot.   Then the next two also began to fade out to nothing but a small dot.  The lights soon became nothing.   The direction they were flying was Westward towards the mountains. I had called several news stations to report and give my sighting to them.  And i also had heard that some people here in Denver, have reported drones flying around out city.   So, I am investigating that story as well.  If this is true, i might have very well have seen night time maneuvers here in Arvada.  I think its still UFO but not ruling out DRONES either.;39.8289982352941;-105.110888235294
Triangle formation to horizontal line.;2017-05-03 02:04:00;3 minutes;Triangle formation to horizontal line. Went outside Friday/Saturday early morning from 12:50AM-2:30 to do my weekend star gazing.  While listening to Coast to Coast AM, at 2:04AM, I noticed, while looking up towards the sky to my NW position, a triangle V formation of lights!  Three pin dot lights of yellow/white color!  With three red colored lights as well!   The lights were in a triangle formation at the start, and ending in a horizontal line formation!  Noticed also three slow blinking red lights at its base!  Object did make a low hum sound as it flew over my backyard!  Moon was also setting west over the front range foothills/mountains.  My first thought was a B-2 or Stealth fighter!  But was too large to be any!  Was bigger than any of our aircraft!  I also investigate as well!   Been a Ufo investigator for 17 years!  As my own private Ufo investigator, this was my first Tirangle/V shape sighting!  As to how high this was,estimate altitude was 7,000ft+.Temperture was 52 degrees out!  Clear skies!!;39.8289982352941;-105.110888235294
I was observing the moon through a telescope looking East. A light to the west caught my attention. I lifted my head away from the eye;2020-04-06 21:01:00;5 seconds;I was observing the moon through a telescope looking East.  A light to the west caught my attention.  I lifted my head away from the eye piece and saw 4 lightly flashing lights shaped like a triangle floating through the sky.  I followed it traveling from southwest to northeast for 5 seconds.  it moved beyond the horizon of the trees in my parking lot.  I heard no sound and I can't tell how far or big it was.  Best guess the size of a hot air balloon with the basket leading north east. I am located at 80th and sheridan in Arvada Colorado 80003 at 21:01.;39.8289982352941;-105.110888235294
