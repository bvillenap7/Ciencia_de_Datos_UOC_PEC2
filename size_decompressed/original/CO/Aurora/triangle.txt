Triangle consisting of 3 red lights. Shot straight up, then stayed in same position for 30 seconds, then finally descended diagonally u;1989-04-18 22:30:00;1 minute;Triangle consisting of 3 red lights.  Shot straight up, then stayed in same position for 30 seconds, then finally descended diagonally until we could no longer see it obstructed by buildings.;39.6695444272445;-104.803991486068
Triangle shaped object traveling slowly over Colorado neighborhood;1998-11-20 20:00:00;5-10 minutes;Triangle shaped object traveling slowly over Colorado neighborhood Some friends and I were snowboarding in the back yard of my neighbors house in the winter and were building a jump off a terraced lawn.  The youngest of us, who then was probably 9 or 10 years of age told us to look at the object flying in the sky.  We told him it was just a plane that was flying low over the neighborhood.  As it got closer we realized he wasn't lying and what he saw wasn't a plane but a triangle shaped object that had no sound to it, or very little because no one paid attention to the sound.  The object looked like a triangle but had smooth corners like that of a thick dorito chip.  We all stood there for a couple minutes staring at the object and it continued to move over our houses.  The object was flying much lower than any plane and had three or four main lights with smaller symmetrical lights between the main ones.  The lighting was not any different color than that of a normal halogen lightbulb, although it was much larger.  I haven't seen one since and none of us have talked about it because we lost touch.  The neighbors I saw it with were all part of the same household and I myself lived next door.  They ended up returning to their hometown in Canada, which is why we haven't talked since.;39.6695444272445;-104.803991486068
Triangular craft (two footbal fields in size);2000-10-26 22:00:00;10 Minutes;Triangular craft (two footbal fields in size) As reported to Art Bell in October 2000, I was fortunate enough to see a Triangular UFO from my home deck in Aurora, CO.  I was out on the deck of my house about 10:00PM looking with binoculars looking out at the city and stars. A huge triangular object at least two football fields in size suddenly appeared from the south above my house.  It was coming from the south direction near NORAD in Colo. Spgs.  It was Triangular in shape, and definitely had cloaking or stealth capability.  It was travelling at a slow rate of speed over my house, and following the I225 corridore parallel to Buckley AFB.  The whole interior of the craft was lighted when decloaked and I could see inside the craft as it passed me.  It had huge rectangular wing spars which may have been power sources or generators.  The control center at the front of the craft was all lighted.  As it was parallel to Buckley AFB (classified-Radar Domes)it started to cloak or go into a stealth mode and as it proceeded North and disappeared.  The amazing part, is Denver International Airport to the north uses this area as a holding pattern for commercial jets landing but, it did not seam to interfere with the Triangular UFO as it slowly moved along the interstate.  I am now retired, and worked for many years as a missile launch technician for the Defense Dept., and as a Quality Manager for McDonnell Douglas building pilot training programs and software, Telecommunications etc.  I am now a firm believer in UFO's, and no debunking by the government or military will ever change my mind.;39.6695444272445;-104.803991486068
Several objects came from big one, then I lost sight of them...;2002-08-03 13:45:00;1-3 minutes;Several objects came from big one, then I lost sight of them... I saw a very large triangle shaped object out my window while ((outside)).  I'm not sure how large, or how far away it was, but I could tell it was moving very slowly.  It seemed to slow down, and stop. then several more, smaller objects seemed to come out of it.  The were circular shaped, and were also moving very slowly.  They all went different directions, and then the bigger one picked up speed (note that it never got smaller, so I could tell that the others came OUT of it), and went the other direction (started going west, then went east).  The others also picked up speed.  I realized that I was peeing all over the floor, and looked down very briefly (about a second, maybe two), and they were gone.  I don't know if there were any other witnesses.  I didn't tell anyone before this, because I have a reputation to uphold, and most of my friends would think I went insane... There didn't seem to be any lights or sounds, but there may have been.  I think it was pretty far away, and it was daytime.;39.6695444272445;-104.803991486068
3 triangle craft,  2 red and one yellow/amber;2008-12-29 22:45:00;3 minutes;3 triangle craft,  2 red and one yellow/amber Strange orange light appeared against the stars in the north east. Seeing that this was a bit strange, I went to get my binoculars after a few minutes of observing this strange light. After retrieving my binoculars, it came closer at a high rate of speed.  As it turned to go west I observed what was orange, now become 2 red objects and one yellow/amber object. The light from all 3 objects did not strobe and the lights appeared as orb/arrowhead shapes in single file one behind the other with two red and one the amber object trailing. As the craft came closer these craft were of triangle shape and all were of equal distance,  but then converged into a triangular formation. The amber was at it's highest point in the triangle formation, and as the objects traveled through the sky, the formation changed to a single file formation and then back to a triangle formation with the amber object now at the bottom of the formation. There was no sound from these craft and there were no aircraft in the area at the time of the sighting.  The speed was enormous for the distance I observed. The objects were about 32-36 thousand feet (perhaps less), descending quickly and were about 3 to 4 miles away. I was able to observe the object for about 6-7 seconds going west. I have a nearly unobstructed 180 degree panoramic view from the back of my house  which points almost directly north. I live in an area that has a lot of airplane traffic, so I know these were not planes. However, soon after, there was a plane that flew over my house and then abruptly changed direction,  the same direction that these craft were headed. My background- I have served in the Air Force for 12 years, and held a Top Secret/SCI clearance and have worked around aircraft during my time in and I was also stationed At Cheyenne Mountain/NORAD for 8 years. I would have sent this in sooner, but I wanted to make a better drawing to send in so I could show the time line and events;39.6695444272445;-104.803991486068
It is raining really bad here and me and my friend were driving to my house and we saw an object in the sky kinda low with 3 lights on;2009-07-21 23:23:00;10-15 seconds;It is raining really bad here and me and my friend were driving to my house and we saw an object in the sky kinda low with 3 lights on it barly moving in the sky then it just dissapered. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  Possible hoax.  PD));39.6695444272445;-104.803991486068
Three green lights in the shape of a triangle moving FAST, sharp turns and dipping low in the sky at Quincy and Piccadilly!!!;2012-07-15 21:20:00;5 minutes;Three green lights in the shape of a triangle moving FAST, sharp turns and dipping low in the sky at Quincy and Piccadilly!!! Three green lights in the shape of a triangle moving VERY FAST, making sharp turns and dipping very low in the sky at Quincy and Piccadilly!!!  Lasted sooo long.;39.6695444272445;-104.803991486068
5-7 white triangles observed flying in formation.;2016-03-29 15:30:00;20-30 seconds;5 to 7 white triangles observed flying in formation. The day was a typical post blizzard, beautiful clear, Colorado day.  I had just pulled into the driveway of my parents' home to visit.  I heard the engines of fighter jets above which drew my attention to the sky.  Now jets overhead is a daily occurrence with Buckley Air Force Base minutes away.  I have become a custom to seeing things in the sky having grown up in Aurora, by the AFB.   At the age of 42, I have come to relize that as young teenager along with a neighbor friend, we witnessed a black triangle almost some 30 years ago.  I digress, back to the recent sighting.   As I was looking for the jets, I noticed a formation of about 5 to 7 white triangles moving south to north over the AFB, but at very high altitude.  To me it did not seem as though the jets were after these objects, but obviously I do not know that for sure.   I observed these triangles for about 20 to 30 seconds as they moved across the sky in formation.  In my mind, I can rule out geese or seagulls as, these objects moved in a tight formation.  One triangle that was on the left of the formation did break from the rest, drop to the rear only to rejoin the formation again on the right side.  It was not long after that I lost sight of the objects.  Again, I know what birds in flock look like and this was not that. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.6695444272445;-104.803991486068
