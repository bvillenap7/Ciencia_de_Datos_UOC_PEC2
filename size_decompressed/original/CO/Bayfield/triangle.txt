Flying triangle in Bayfield, CO;2003-10-02 21:00:00;5 min;Flying triangle in Bayfield, CO Okay, this is kind of cheesy beacuse I didnt actually see the thing my father did, but there is no doubt in my mind that he really did.  We live in a very small town, and my father had to dive my mother to a hospital in nearby Durango.  On his return he pulled to the side of the road to observe a triangular craft with blinking red and blue lights underneath, and making no sound as it passed closely over his car.  My father would never lie to me, especially on such a dramatic night, with my mother's condition and all.  So why am I, a non-witness telling you this ten months after the fact?  I actually forgot about what he told me until I was watching some show on TV about aliens and they showed a triangular craft, reminding me.  I went on the internet to research it and came across this site.  Thought I would go ahead and report the sighting for him, because my dad would never do such a thing, and I just thought it might be helpful.;37.261;-107.6068
