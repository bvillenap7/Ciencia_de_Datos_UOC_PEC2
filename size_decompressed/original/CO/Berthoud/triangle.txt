HUGE red and golden triangle object-later photographed & witnessed by a third-party in Salida, CO;2014-09-25 19:00:00;1 minute;HUGE red and golden triangle object  - later photographed & witnessed by a third-party in Salida, CO My UFO sighting took place several days after the Equinox (Sept, 2014) at sunset.  It was a very clear evening sky over the Rocky Mountain’s front range.  From my home in Berthoud, Colorado, I have unobstructed view of Longs Peak and the front range.  I watched the sunset just to the left of Longs Peak.  It was a beautiful golden sunset.   Suddenly, there was a “large laser beam of light” that came from the direction of the sunset and as it approached overhead, somewhat to the right of where I was standing, appeared a huge triangle shaped object.  It was not metallic looking.  The UFO was high in the sky at cloud level.  The object’s design was akin to ‘tinker toy’.   The three disc connectors were red and they were HUGE.  The connector pipes were long cylindrical golden tubes.  I could see clear sky inside and up through the triangle shaped object.  I froze and blinked my eyes because I could not believe what I was seeing.  I mean, I purposefully blinked. When I ope! ned my eyes, I made mental note of the red round disc connectors.  All I can compare them to is extremely large round 'red throat lazengers'.  They were kind of translucent looking but yet a solid shape.  The huge connecting tubes were like pure golden color and very long.  All three tubes were the same length – a great distance of length between the three red disc connectors.  I was stunned. I blinked, again on purpose, and there it was. It was then I thought the disc connectors looked like 'throat lazengers tablets'.  I could see no openings in the forward most red disc connector.  The thing was huge.  Very colorful. It made no sound.  I then blinked one more time and when I opened my eyes the UFO was gone.  Afterwards, I had a sense that I saw three beams and jettisoned object(s) out of the sunset.   When they got in front of my house they came to rest momentarily and then vanished.  Each beam from the sunset had the same trajectory and the UFO’s came to rest in the same spot,  high up and in front of the west side of house.  The object was five times the palm of my hand (though I didn’t perform this step in my observations).  All I can say the UFO was extremely large.  Each of red disc of the triangle was the size of an aircraft carrier, high up in the sky at cloud level.  I was confused and in shock.  It all happened in less than a minute.   I am 68 years old.  I study UFO reports, and have done so since I was a kid.  I was so confounded by what I saw it didn’t even mention it to my wife.  I told no one about it until January 2015, and again to a group of friends in Feb, 2015, when the subject of UFOs came up.   Just recently, May 29, 2015, I found this image: http://nationalufocenter.com/file/2015/04/COSedalia24Apr151.png  ...on George Filer's web site.  I couldn’t believe it.  The MUFON picture (April 24, 2015) is very close to what I saw and it was taken in Sedalia, Colorado, during a sunset!   Background:  I am a NASA life science researcher and reference to my work can be found on NASA's web site.  I've lived Berthoud, Colorado for 29 years.;40.2993;-105.1055
