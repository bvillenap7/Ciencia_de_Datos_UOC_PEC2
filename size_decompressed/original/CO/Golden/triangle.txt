Rapidly moving V-shaped black object, I detected no noise from object moving south.;2014-10-20 22:00:00;15 seconds;Rapidly moving &quotV&quot shaped black object, I detected no noise from object moving south. V shaped craft, silent and moving rapidly, two rows of dim white lights in shape of V (bottom of craft presumably)  No red lights or blinkers like on commercial flights.  Barely seen against night sky, black or grey in color.  I was out looking for Orionids, lots of airplane traffic, but this snuck in just west of me going south along front range between Golden and mountains (maybe 20,000 feet, not too high, lower than all commercial flights I witnessed).  You see it, try and make sense of what it could be, then realize it made no noise whatsoever and was moving fast.  I had once seen an unexplainable pattern of lights, a long time ago and experienced the missing time, so when I went in last night I checked the clock first thing, so unusual amount of time had past like the lst time. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.7598265306123;-105.220002040816
